src/AlpineSkiHouse.Web/Models/Scan.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AlpineSkiHouse.Models
+{
+    public class Scan
+    {
+        public int Id { get; set; }
+
+        public int CardId { get; set; }
```

Comment:
Am I right in thinking that a card can have only one active pass at a time?

What if I have an annual pass for the small hill, where I'm staying, and at noon I decide to trot down the road to the other hill on a day pass?  

Are we worried about that use case? Might be too edge for the topic of our book.